extensions/blocks/contact-info/class-jetpack-contact-info-block.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Jetpack_Contact_Info_Block
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Helper class that lets us add schema attributes dynamically because they are not something that is store with the content.
+ * Due to the limitations of wp_kses.
+ *
+ * @since 7.1.0
+ */
+class Jetpack_Contact_Info_Block {
+
+	/**
+	 * Adds contact info schema attributes.
+	 *
+	 * @param array  $attr    Array containing the contact info block attributes.
+	 * @param string $content String containing the contact info block content.
+	 *
+	 * @return string
+	 */
+	public static function render( $attr, $content ) {
+		Jetpack_Gutenberg::load_assets_as_required( 'contact-info' );
+
+		return str_replace(
+			'class="wp-block-jetpack-contact-info"',
+			'class="wp-block-jetpack-contact-info" itemprop="location" itemscope itemtype="http://schema.org/Organization"',
+			$content
+		);
+	}
+
+	/**
+	 * Adds address schema attributes.
+	 *
+	 * @param array  $attr    Array containing the address block attributes.
+	 * @param string $content String containing the address block content.
+	 *
+	 * @return string
+	 */
+	public static function render_address( $attr, $content ) {
+		// Returns empty content if the only attribute set is linkToGoogleMaps.
+		if ( ! self::has_attributes( $attr, array( 'linkToGoogleMaps' ) ) ) {
```

Comment:
You'll prolly need to omit `className` here?

Reply:
Great catch! added the className to be ommited for all the blocks that support the className attribute.